src/tile/FeaturedTile.js

```diff
@@ -0,0 +1,133 @@
+import React, { PropTypes } from 'react';
+import {
+	TouchableOpacity,
+	View,
```

Comment:
Not sure why my editor does this. Maybe think about adding an [editorconfig](!http://editorconfig.org/)?